x/bep3/keeper/params.go

```diff
@@ -18,63 +18,100 @@ func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
 	k.paramSubspace.SetParamSet(ctx, &params)
 }
 
-// GetBnbDeputyAddress returns the Bnbchain's deputy address
-func (k Keeper) GetBnbDeputyAddress(ctx sdk.Context) sdk.AccAddress {
+// ------------------------------------------
+//				Asset
+// ------------------------------------------
+
+// GetAsset returns the asset param associated with the input denom
+func (k Keeper) GetAsset(ctx sdk.Context, denom string) (types.AssetParam, error) {
 	params := k.GetParams(ctx)
-	return params.BnbDeputyAddress
+	for _, asset := range params.AssetParams {
+		if denom == asset.Denom {
+			return asset, nil
+		}
+	}
+	return types.AssetParam{}, sdkerrors.Wrap(types.ErrAssetNotSupported, denom)
 }
 
-// GetBnbDeputyFixedFee returns the deputy's fixed fee
-func (k Keeper) GetBnbDeputyFixedFee(ctx sdk.Context) sdk.Int {
+// SetAsset sets an asset in the params
+func (k Keeper) SetAsset(ctx sdk.Context, asset types.AssetParam) {
 	params := k.GetParams(ctx)
-	return params.BnbDeputyFixedFee
+	for i := range params.AssetParams {
+		if params.AssetParams[i].Denom == asset.Denom {
+			params.AssetParams[i] = asset
+		}
```

Comment:
Is this a requirement? I think if, for example, we update the bep3 params via governance, it will just call `k.SetParams()` which will override the whole params.